service/chartconfig/v1/helm/error.go

```diff
@@ -22,6 +22,10 @@ var releaseNotFoundError = microerror.New("release not found")
 
 // IsReleaseNotFound asserts releaseNotFoundError.
 func IsReleaseNotFound(err error) bool {
+	if err == nil {
+		return false
+	}
```

Comment:
The default below is already false. No need for this here.

Reply:
The problem is if the error is nil. If I remove it then it needs this in the API calls.

```
	resp, err := c.helmClient.ReleaseContent(releaseName)
	if err != nil && IsReleaseNotFound(err) {
		return nil, microerror.Maskf(releaseNotFoundError, releaseName)
	} else if err != nil {
		return nil, microerror.Mask(err)
	}
```

I think its cleaner to handle it in the error matcher. Going with this but let me know if you disagree.